Pull

SpiralDJ opened the pull request.

SpiralDJ committed:
Added monitor speed and monitor filter settings
To configure serial input.

Speed set to 115200 Baud
Set to send serial input on enter

SpiralDJ committed:
Changed line 145 of rf24.h from protected to
public for access to readRegister funcion

Created test code for nrf24

vinelopre reviewed (commented):
Included initial nRF transmission programs.

vinelopre reviewed (approved):
Changed nRF class to public.

vinelopre reviewed (approved):
Edited `uno_main.cpp`

SpiralDJ committed:
comment

vinelopre committed:
Merge remote-tracking branch 'origin/main' into pr/SpiralDJ/34

Merged with SpiralDJ's repository.

SpiralDJ committed:
Changed line 39 of Print.h from private to public
to access Serial.printFloat

Serial.println is printing the integer array in
binary. I dont know why.

Fixed nrf write test. It works as long as another
arduino with an nrf module is recieving

SpiralDJ committed:
Merge branch 'main' of https://github.com/SpiralDJ/BCC-Anthropomorphic-Robotic-Arm

vinelopre committed:
Modified nRF branch source code and placed it in a
separate library.

vinelopre committed:
Edited RF24.h to have a protected class.

vinelopre committed:
Merge branch 'main' of https://github.com/spiraldj/bcc-anthropomorphic-robotic-arm into pr/SpiralDJ/34
